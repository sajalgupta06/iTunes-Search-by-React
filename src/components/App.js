import React from 'react';
import Header from './Header';
import Container from './Container';
import '../style/App.css';

const App = () => {
  return (
    <div>
      <Header />
      <Container />
     
    </div>
  );
};

export default App;
