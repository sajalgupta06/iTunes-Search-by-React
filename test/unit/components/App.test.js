import React from 'react';
import { shallow } from 'enzyme';
import App from '../../../src/components/App';

describe('<App />', () => {
  test('App component should renders correctly', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
