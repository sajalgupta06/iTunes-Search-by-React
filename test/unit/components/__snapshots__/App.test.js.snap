// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> App component should renders correctly 1`] = `
<div>
  <Header />
  <Container />
  <Ribbon />
</div>
`;
